AndelaOSP/andela-societies-backend
# [WIP Chore] Restructure logged activities module

## Resolves #WIP

## Description

  -
  -

## Fix

  - Update requirements
  - Fix production seeding test data bug
  - Update activity types 'supports_multiple_participants' field type to
boolean

## How to test

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.
- Ensure you have `rabbitmq` running and run `python run_tests.py` at the root of the `src` directory
